src/systemctl/systemctl-start-unit.c

```diff
@@ -298,6 +299,17 @@ int verb_start(int argc, char *argv[], void *userdata) {
                                 job_type = "start";
                                 mode = "isolate";
                                 suffix = ".target";
+                        } else if (streq(argv[0], "toggle")) {
+                                /* A "systemctl toggle <unit1> <unit2> …" command */
+                                r = verb_is_active(argc, argv, userdata);
```

Comment:
If unit1 is active and unit2 is inactive, then this tries to stop both unit1 and unit2... But, this should stop unit1 and start unit2 in that case.

Also, `verb_is_active()` -> `check_unit_generic()` prints unit state when `arg_quiet` is false. The output should not be printed here.

Reply:
Yes. I'll fix them an update.